addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/GitSteps.java

```diff
@@ -83,8 +84,7 @@ public void pushToGitRepository(OsioLaunchProjectile projectile, GitRepository r
     public void createWebHooks(OsioProjectile projectile, GitRepository gitRepository) {
         String jenkinsWebhookURL = EnvironmentVariables.ExternalServices.getJenkinsWebhookURL();
         try {
-            // TODO: Check if the webhook requires a secret
-            gitService.createHook(gitRepository, null, new URL(jenkinsWebhookURL));
+            gitService.createHook(gitRepository, UUID.randomUUID().toString(), new URL(jenkinsWebhookURL));
```

Comment:
The secret was a random UUID? That looks really weird

Reply:
was wondering about that as well but yeah look here:
https://github.com/edewit/launcher-backend/blob/56f0e000bee6818b6fcd63c62cd19727260dbae6/addons/legacy-osio-addon/src/main/java/io/fabric8/forge/generator/kubernetes/CreateBuildConfigStep.java#L574
and `useUiidForBotSecret` is always `true`